zanata-war/src/test/java/org/zanata/file/TranslationDocumentUploadTest.java

```diff
@@ -23,14 +23,13 @@
 import static javax.ws.rs.core.Response.Status.NOT_ACCEPTABLE;
 import static org.mockito.Mockito.doThrow;
 
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
 import org.mockito.Mock;
 import org.mockito.MockitoAnnotations;
-import org.testng.annotations.AfterMethod;
-import org.testng.annotations.BeforeMethod;
-import org.testng.annotations.Test;
 import org.zanata.exception.ChunkUploadException;
 
-@Test(groups = { "unit-tests" })
```

Comment:
In general, I guess we are losing test groups. I take it we were not using them.